x-pack/plugins/cases/docs/openapi/paths/s@{spaceid}@api@cases@{caseid}@user_actions.yaml

```diff
@@ -1,7 +1,7 @@
 get:
   summary: Returns all user activity for a case.
   description: >
-    You must have `read` privileges for the **Cases** feature in the
+    Deprecated in 8.1.0. Use the find user actions API instead. You must have `read` privileges for the **Cases** feature in the
```

Comment:
Should add that it will be removed in a future release like the others?

Reply:
Yup, can do. Do you also have advice for what the alternative recommendation should be for the "get case status" API? Should they seek that info in the "find cases" output instead?